src/main/java/com/amazon/opendistroforelasticsearch/security/privileges/PrivilegesEvaluator.java

```diff
@@ -94,6 +95,23 @@
 public class PrivilegesEvaluator implements DCFListener {
 
 
+    final String[] blockIndexActionPatternsList = new ArrayList<String>() {
+        {
+            add("indices:data/write*");
+            add("indices:data/read*");
+            add("indices:admin/delete*");
+            add("indices:admin/create*");
+            add("indices:admin/mapping/delete*");
+            add("indices:admin/mapping/put*");
+            add("indices:admin/freeze*");
+            add("indices:admin/settings/update*");
+            add("indices:admin/aliases");
+            add("indices:admin/close*");
+            // Todo decide if we want this index blocked during snapshots
+//            add("indices:admin/snapshot/restore*");
```

Comment:
behavior of snapshot for this index should be similar as .opendistro_security index (admin with admin cert)